Add riscv-rt to workspace

romancardenas opened the pull request:
This PR adds [`riscv-rt`](https://github.com/rust-embedded/riscv-rt) to this workspace. I'm still working on fine-tuning the CI, let me know your opinion and I'll push it to the PR 😄

romancardenas commented:
@rust-embedded/riscv good to go! Take a look and let me know if I forgot something

jessebraham commented on riscv-rt/README.md:
```diff
@@ -0,0 +1,41 @@
+[![crates.io](https://img.shields.io/crates/d/riscv-rt.svg)](https://crates.io/crates/riscv-rt)
+[![crates.io](https://img.shields.io/crates/v/riscv-rt.svg)](https://crates.io/crates/riscv-rt)
+[![Build Status](https://travis-ci.org/rust-embedded/riscv-rt.svg?branch=master)](https://travis-ci.org/rust-embedded/riscv-rt)
```
Is Travis CI actually used? Doesn't appear to be, I'm guessing this is just an artifact from the past.

jessebraham commented on riscv-rt/macros/Cargo.toml:
```diff
@@ -0,0 +1,24 @@
+keywords = ["riscv", "runtime", "startup"]
+license = "MIT OR Apache-2.0"
+name = "riscv-rt-macros"
+repository = "https://github.com/rust-embedded/riscv-rt"
```
```suggestion
repository = "https://github.com/rust-embedded/riscv"
```

jessebraham commented on riscv/README.md:
```diff
@@ -0,0 +1,41 @@
+[![crates.io](https://img.shields.io/crates/d/riscv.svg)](https://crates.io/crates/riscv)
+[![crates.io](https://img.shields.io/crates/v/riscv.svg)](https://crates.io/crates/riscv)
+[![Build Status](https://travis-ci.org/rust-embedded/riscv.svg?branch=master)](https://travis-ci.org/rust-embedded/riscv)
```
See above.

jessebraham reviewed (changes requested):
Few little nitpicks but otherwise LGTM!

romancardenas commented:
Thanks for your review @jessebraham ! I addressed all your comments and adapted the general `README.md` file from [`cortex-m`](https://github.com/rust-embedded/cortex-m)

jessebraham reviewed (approved):
Thanks for making those changes, LGTM!
